enhancements/scheduling/scheduler-profiles.md

```diff
@@ -0,0 +1,212 @@
+---
+title: scheduling-profiles
+authors:
+  - "@damemi"
+reviewers:
+  - "@soltysh"
+  - "@ingvagabund"
+  - "@deads2k"
+approvers:
+  - "@soltysh"
+  - "@ingvagabund"
+  - "@deads2k"
+creation-date: 2020-11-11
+last-updated: 2020-11-11
+status: provisional
+see-also:
+  - "https://github.com/kubernetes/enhancements/blob/master/keps/sig-scheduling/624-scheduling-framework/kep.yaml"
+  - "/enhancements/kube-apiserver/audit-policy.md"
+replaces:
+superseded-by:
+---
+
+# Scheduling Profiles
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement proposes a new field in the `schedulers.config.openshift.io/v1` API to allow
+configuration of enabled and disabled scheduler plugins via defined profiles.
+
+This will simplify how users configure pod scheduling beyond the default set of enabled plugins in the
+default scheduler managed by the OpenShift operator. 
+
+## Motivation
+
+As of Kubernetes 1.19, plugin profile configuration has replaced Policy files for the scheduler.
+We currently support setting scheduler predicates and priorities through the Policy API, and need
+to offer similar configuration with the new API in order to maintain a level of feature parity.
+
+However, the new plugin API is more expansive than the old API and provides more configuration possibilities
+than is necessary for most customers or reasonable to support. For those reasons we propose an
+abstracted representation of the most common plugin configurations for users to configure.
+
+### Goals
+
+1. define several scheduler plugin configurations that represent the most common use-cases
+2. add a `profile` API field to `schedulers.config.openshift.io/v1` to specify which predetermined
+profile to use
+3. enable these profiles by kube-scheduler-operator
+
+### Non-Goals
+
+* fine configuration of individual scheduler plugins
+  * This differs from the current scheduler Policy config, which allows individual control over predicates and priorities
+* default support for custom schedulers or out-of-tree plugins
+
+Note that these non-goals are only for the default cluster scheduler managed by OpenShift, and should not imply any inherent platform restrictions on secondary or other custom schedulers users choose to deploy on their own.
+
+## Proposal
+
+This enhancement proposes a new field in `schedulers.config.openshift.io/v1` named `profile`. This field
+specifies which of the following plugin configurations (as scheduler profiles to enable on the default cluster
+scheduler.
+
+* `LowUtilization`: this will use the default scheduler profile as defined in the kube-scheduler's internal registry.
```

Comment:
lead with the goal of the profile.  Is this `LowNodeUtilization`, where I want to spread load evenly across a pool of nodes to avoid overloading any individual node?